packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -229,8 +235,7 @@ function useBlockEditorSettings( settings, postType, postId ) {
 			// Check these two properties: they were not present in the site editor.
 			__experimentalCreatePageEntity: createPageEntity,
 			__experimentalUserCanCreatePages: userCanCreatePages,
-			pageOnFront,
-			pageForPosts,
```

Comment:
I think these settings were meant to be experimental. I think we should be ok with removing it, thinks will just keep working fine with `undefined`.

Reply:
I think you are right 🤞